js/pluralizer.js

```diff
@@ -0,0 +1,4 @@
+let thing = 'console';
+let count = 6;
+
```

Comment:
You'll need an if statement to account for if you have ONE single thing or MANY things.